tests/settings.py

```diff
@@ -28,16 +29,15 @@ def local_path(path):
     'tests.tests'
 ]
 
-MIDDLEWARE_CLASSES = (
-    'django.contrib.sessions.middleware.SessionMiddleware',
-    'django.contrib.auth.middleware.AuthenticationMiddleware'
-)
 
 ROOT_URLCONF = 'tests.urls'
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = MIDDLEWARE_CLASSES = (
```

Comment:
MIDDLEWARE_CLASSES isn't needed anymore. also, use [ ] instead of ()

Reply:
addressed. how should we deal with docs / breaking compat change?